src/index.js

```diff
@@ -16,4 +16,4 @@
 
 export * from './useFetchye';
 export * from './makeServerFetchye';
-export { FetchyeReduxProvider } from './FetchyeContext';
+export * from './FetchyeProvider';
```

Comment:
Any reason not to use named exports? This could potentially expose APIs that were not meant to be in the API contract.